contentcuration/contentcuration/static/js/edit_channel/uploader/views.js

```diff
@@ -133,10 +133,9 @@ var EditMetadataView = BaseViews.BaseEditableListView.extend({
   load_editor:function(selected_items){
     var is_individual = selected_items.length === 1;
     var is_exercise = is_individual && selected_items[0].model.get("kind") == "exercise";
-    var has_files = is_individual && (selected_items[0].model.get("assessment_items").length ||
-                    selected_items[0].model.get("files").find(function(f){
-                      return window.formatpresets.get({id:f.preset.name || f.preset.id || f.preset}).get("display");
-                    }));
+    var has_files = is_individual && selected_items[0].model.get("files").some(function(f){
```

Comment:
I would have broken this up into a few more lines/variables to make it easier to read. Not actually sure how this is ensuring that it has files.

Also, is there a reason we're only checking the first selected item?